nkms/blockchain/eth/sol/compile.py

```diff
@@ -48,7 +48,7 @@ def compile(self) -> dict:
             sol_contract_paths.extend(glob.iglob(source_dir + '/**/*.sol', recursive=True))
 
         remapping_dirs = ["contracts={}".format(self._contract_source_dirs[0])]
-        compiled_sol = compile_files(sol_contract_paths, import_remappings=remapping_dirs)
+        compiled_sol = compile_files(sol_contract_paths, import_remappings=remapping_dirs, optimize=True)
```

Comment:
What effect does optimize have on the compiler (Does this relate to bytecode size?)?

Reply:
First of all it reduces size of bytecode but we should find suitable number of steps (200 by default). Also it should optimize bytecode to using less gas